torch/autograd/_functions/basic_ops.py

```diff
@@ -169,6 +169,14 @@ def backward(ctx, grad_output):
 
 @traceable
 class MulConstant(InplaceFunction):
+    @staticmethod
+    def symbolic(g, a, b, inplace=False):
+        assert not inplace, "Exporting inplace MulConstant is not supported yet"
```

Comment:
We are never supporting inplace MulConstant, so this line should just be deleted.